backend/osmosewebsite/tests/views/trap.py

```diff
@@ -0,0 +1,81 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+from backend.osmosewebsite.serializers.trap import TrapFields
+
+
+class TrapViewSetTestCase(APITestCase):
+    """Test TrapViewSetTestCase when list or detail trap are request"""
+
+    fixtures = ["users", "trap"]
+    creation_data = {
+        "lastname": "string",
+        "firstname": "string",
+        "title": "string",
+        "intro": "string",
+        "body": "string",
+        "date": "2022-01-25T10:42:15Z",
+        "thumbnail": "string",
+    }
+
+
+def test_list(self):
```

Comment:
The test isn't executed.
Be really careful with the indent in python. Here the function is outside the class, so it won't be executed.